containers/runner/README.md

```diff
@@ -20,61 +20,43 @@ There is a [playbook](runner-host.yml) for deployment of the host on Fedora Clou
 
 Run a test in a container
 -------------------------
-
-Download the [automatically built](.github/workflows/container-autoupdate.yml) official container image:
+Use the [launch](./launch) script to run a set of tests from the current kickstart-tests directory in the runner container:
 ```
-podman pull quay.io/rhinstaller/kstest-runner
+containers/runner/launch keyboard [test2 test3 ...]
 ```
 
-Or build the container yourself:
+This will download the [automatically built](.github/workflows/container-autoupdate.yml) [official container image](https://quay.io/repository/rhinstaller/kstest-runner).
+
+You can also build the container yourself to test modifications to it:
 ```
 podman build -t rhinstaller/kstest-runner .
 ```
 
-Define the directory for passing of data between the container and the system (via volume):
-```
-export VOLUME_DIR=${PWD}/data
-```
+The `launch` script creates a `./data/` directory for passing of data between the container and the system (via volume). By default it downloads the current Fedora Rawhide boot.iso, but to test some other image you can put it into `data/images/boot.iso` before running `launch`.
 
-Create subdirs for test inputs (installation image) and outputs (logs):
+The result logss get written into `./data/logs/`:
```

Comment:
`The result logs ...`

Reply:
No, this is parseltongue, totally correct!

:stuck_out_tongue_winking_eye: thanks for spotting, fixed.

Reply:
:snake: :snake: :snake: